scripts/prep_local_devel_env.sh

```diff
@@ -180,8 +180,16 @@ echo "Sleeping for a few seconds to avoid issues with broker not being able to t
 echo "Appears like there is a delay of when we create the asb-etcd route and when it is available for use"
 sleep 5
 
-etcd_route=`oc get route asb-etcd -n ${ASB_PROJECT} -o=jsonpath=\'\{.spec.host\}\'`
-echo "etcd route is at: ${etcd_route}"
+if [ -z "$LOCAL_ETCD" ]; then
```

Comment:
This doesn't do what you want.
```
      -z string
              True if the length of string is zero.
```

```
LOCAL_ETCD ="false"
LOCAL_ETCD="true"
```
will both result in using localhost.

The only way to use the `oc get route` version is if LOCAL_ETCD is unset entirely.


### Empty
```
$ ./z.sh          
LOCAL_ETCD =                                  
using oc get route and port 80                
```
### set to true
```
$ ./z.sh true     
LOCAL_ETCD = true                             
using localhost etcd on 2379                  

### set to false
```
$ ./z.sh false    
LOCAL_ETCD = false                            
using localhost etcd on 2379 
```